pep-0458.txt

```diff
@@ -51,7 +51,10 @@ would need to steal multiple signing keys, which are stored independently,
 in order to compromise the role responsible for specifying a repository's available
 files.  Or, alternatively, a role
 responsible for indicating the latest snapshot of the repository may also have to be
-compromised.
+compromised. More background about TUF and information about other TUF
```

Comment:
I don't think this is really background information, no? Perhaps better to just add it as another link for TUF when it is first defined?